sormas-backend/doc/UserDataAccess.md

```diff
@@ -1,62 +1,131 @@
-# Basics
+# User data access
+
+## Basics
 In general data access & synchronisation is based on the following rules:
 
 * hospital informant: all data associated to the facility (cases)
 * community informant: all data associated to the community (cases)
 * officers: all data associated to the district (cases, events)
 * supervisors: all data associated to the region
+* observers: all data associated to their jurisdiction (nation, region or district)
+* port of entry users: all data associated to their jurisdiction (nation, region or district)
```

Comment:
Port of entry -> Port health

Also, some point of entry users always have a point of entry assigned, so it's basically them being able to view data associated to their point of entry; only national point of entry users see data from multiple points of entry

Reply:
please check this again, how i extended